components/StepsSlider/index.tsx

```diff
@@ -1,51 +1,71 @@
 import { useEffect, useRef, useState } from 'react';
 import { Button } from 'react-bootstrap';
 import Carousel from 'react-bootstrap/Carousel';
+import NodeBox from '../NodeBox/NodeBox';
 import Step from '../Step';
 
 // TODO: Remove any
-function StepsSlider({steps} : {steps: any}) {
+function StepsSlider({ visualization }: { visualization: any }) {
```

Comment:
please declare types wherever there's any
it is a good ocasion for you to learn TS

Reply:
@danielpopek94